spec/jobs/sufia_resque_queue_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
+
+describe Sufia::Resque::Queue do
+  let (:subject) { Sufia::Resque::Queue.new "test_queue" }
+  let (:job) { double('job') }
+
+  context "no errors" do
+    it "queues the job" do
+      expect(::Resque).to receive(:enqueue_to).and_return(true)
+      subject.push(job)
+    end
+  end
+
+  context "timeout error" do
+    it "retries on timeout" do
+      expect(::Resque).to receive(:enqueue_to).and_raise(Redis::TimeoutError)
```

Comment:
@mjgiarlo check out the rspec readme https://github.com/rspec/rspec-mocks/blob/master/README.md#setting-responses
There is an example of the syntax I used there...